app/components/trainees/confirmation/confirm_publish_course/view.rb

```diff
@@ -32,35 +33,24 @@ def rows
               value: course_details,
               action: govuk_link_to(t(".change_course"), edit_trainee_publish_course_details_path(@trainee)),
             },
-            {
-              key: t(".subject"),
-              value: subject,
-            },
-            {
-              key: t(".level"),
-              value: level,
-            },
-            {
-              key: t(".age_range"),
-              value: age_range,
-            },
-            {
-              key: t(".start_date"),
-              value: start_date,
-            },
-            {
-              key: t(".duration"),
-              value: duration,
-            },
+            { key: t(".subject"), value: subject_names },
```

Comment:
Should the subject heading be pluralised when multiple subjects are present?
similar to https://github.com/DFE-Digital/register-trainee-teachers/blob/08b2f044ae438dfc4b0ce91a525d4046b517f3fb/app/components/trainees/confirmation/course_details/view.html.erb#L15 (seems to have a regression in master, hence linking an older commit)

Reply:
Have resolved this in https://github.com/DFE-Digital/register-trainee-teachers/pull/940/commits/d5e7671f3039c02be748bbb4488a78ffe32d77b9